plugins/org.jkiss.dbeaver.model.ai/src/org/jkiss/dbeaver/model/ai/impl/AIAssistantImpl.java

```diff
@@ -171,17 +171,14 @@ public AICommandResult command(
             engine.getContextWindowSize(monitor)
         );
 
-        List<AIMessage> chatMessages = List.of(
-            AIMessage.systemMessage(prompt),
-            AIMessage.userMessage(request.text())
-        );
-
         AIEngineResponse completionResponse = requestCompletion(engine, monitor, completionRequest);
-
+        String responseVariant = completionResponse.variants().stream().findFirst().orElseThrow(
+            () -> new DBException("Empty AI response when executing command")
+        );
```

Comment:
Let's prettify it a bit:
```suggestion
        String responseVariant = completionResponse.variants().stream()
            .findFirst()
            .orElseThrow(() -> new DBException("Empty AI response when executing command"));
```